sdk/storage/azure-storage-queue/tests/queuetestcase.py

```diff
@@ -307,96 +146,6 @@ def assertNamedItemNotInContainer(self, container, item_name, msg=None):
                     repr(item_name), repr(container))
                 self.fail(self._formatMessage(msg, standardMsg))
```

Comment:
Now that we're not using `unittest.Testcase`, I don't think `self.fail` will work any more.
There should be a pytest equivalent - maybe even just `pytest.fail`

Reply:
yup..it's pytest.fail :)